plugins/urls/cookiejar.py

```diff
@@ -0,0 +1,52 @@
+__author__ = 'Gareth Coles'
+
+from cookielib import LWPCookieJar
+
+
+class ChocolateCookieJar(LWPCookieJar):
```

Comment:
Any reason for the name other than liking chocolate cookies? I feel like `CookieJar` would be more appropriate, although it's not really important since it's only used internally.